test/unittests/NDB_BVL_Instrument_LINST_ToJSON_Test.php

```diff
@@ -35,6 +35,9 @@ function setUp() {
 
         $factory->setDatabase($mockdb);
         $factory->setConfig($mockconfig);
+        $mockdb->expects($this->any())
+               ->method('pselectOne')
+               ->willReturn('999');
```

Comment:
line 52         $this->i->method('getSessionID')->willReturn(123456);
                     change to $this->i->method('getSessionID')->willReturn(new \SessionID(strval("123456")));

Reply:
@kongtiaowang Done.